draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -728,56 +728,81 @@ If any step fails, the Presentation is not valid and processing MUST be aborted.
 
-# Enveloping SD-JWTs {#enveloping}
-
-In some applications or transport protocols, it is desirable to put an SD-JWT into an outer JWT container. For example, an implementation may envelope multiple credentials and presentations, independent of their format, in a JWT to enable application-layer encryption during transport.
-
-For such use cases, the SD-JWT SHOULD be transported as a single string. Key Binding MAY be achieved by signing the envelope JWT instead of including a separate Key Binding JWT in the SD-JWT.
-
-The following non-normative example shows an SD-JWT Presentation enveloped in a JWT:
-
-```
-{
-  "aud": "https://verifier.example.org",
-  "iat": 1580000000,
-  "nonce": "iRnRdKuu1AtLM4ltc16by2XF0accSeutUescRw6BWC14",
-  "_sd_jwt": "eyJhbGci...emhlaUJhZzBZ~eyJhb...dYALCGg~"
-}
-```
-
-Here, the SD-JWT is shown as the value of an `_sd_jwt` claim where `eyJhbGci...emhlaUJhZzBZ` represents the Issuer-signed JWT and `eyJhb...dYALCGg` represents a Disclosure. The SD-JWT does not contain a Key Binding JWT as the outer container can be signed instead.
-
-Other specifications or profiles of this specification may define alternative formats for transporting an SD-JWT that envelope multiple such objects into one object and provide Key Binding using means other than the Key Binding JWT.
-
 # JWS JSON Serialization {#json_serialization}
 
 This section describes an optional alternate format for SD-JWT using the JWS JSON Serialization from [@!RFC7515].
 
 For both the General and Flattened JSON Serialization, the SD-JWT is represented as a JSON object according
-to Section 7.2 of [@!RFC7515]. The disclosures (both for issuance and presentation) are included in the
-serialized JWS using the key `disclosures` at the top-level of the JSON object (the same level as the `payload` member). The
+to Section 7.2 of [@!RFC7515]. The disclosures (both for issuance and presentation) SHOULD be included in the
+serialized JWS using the member name `disclosures` at the top-level of the JSON object (the same level as the `payload` member). The
 value of the `disclosures` member is an array of strings where each element is an individual Disclosure
 as described in (#creating_disclosures). The Issuer includes a Disclosure for each selectively
 disclosable claim of the SD-JWT payload, whereas the Holder includes only the Disclosures
-selected for the given presentation. Additionally, for presentation with a Key Binding, the Holder adds
-the key `kb_jwt` at the top-level of the serialized JWS with a string value containing the
-Key Binding JWT as described in (#kb-jwt).
+selected for the given presentation.
+
+Alternative methods for conveying the disclosures MAY be used (such as including them in a `disclosures`
+member of an outer JSON structure also containing the JSON Serialized SD-JWT) as dictated by a specific
+application or transport protocol. However, the details of such approaches fall outside the scope of this
```

Comment:
Maybe something like "should be defined by the profile of this specification" would be more actionable?

Reply:
If they define something that is outside of this specification, it is not a profile. I think the current wording is fine.